src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/attn_memcpy.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <float.h>
+
+#include <cmath>
+#include <cstring>
+#include <iostream>
+#include <limits>
+#include <type_traits>
+
+#if defined(HAVE_AVX2) || defined(HAVE_AVX512F)
+#    include <immintrin.h>
+#endif
+
+#include "openvino/core/type/bfloat16.hpp"
+#include "openvino/core/parallel.hpp"
+#include "common.hpp"
+#include "attn_memcpy.hpp"
+
+namespace InferenceEngine {
+namespace Extensions {
+namespace Cpu {
+namespace XARCH {
+
+using namespace ov;
+
+// float16 <- float
+template<typename TA, typename TB>
+void attn_copy(TA* a, TB* b, size_t n) {
+    size_t i = 0;
+#if defined(HAVE_AVX512F)
+    for (; i + vec_len_f32_avx512 <= n; i += vec_len_f32_avx512) {
+        auto vb = mm512_uni_loadu_ps(b + i);
+        mm512_uni_storeu_ps(a + i, vb);
+    }
+#elif defined(HAVE_AVX2)
+    for (; i + vec_len_f32_avx2 <= n; i += vec_len_f32_avx2) {
+        auto vb = mm256_uni_loadu_ps(b + i);
+        mm256_uni_storeu_ps(a + i, vb);
+    }
+#endif
+    for (; i < n; i++) {
```

Comment:
may use mask to avoid tails cost

Reply:
Such as intrinsic `_mm256_maskz_loadu_epi16` needs compiler option `-mavx512bw -mavx512vl` etc, which currently set to `-mavx512f`. In actual model the head size is typically 64 or 80 or 128, tail is not common, leave them unchanged.